table/partitioned_filter_block.cc

```diff
@@ -109,12 +119,14 @@ PartitionedFilterBlockReader::PartitionedFilterBlockReader(
     const SliceTransform* prefix_extractor, bool _whole_key_filtering,
     BlockContents&& contents, FilterBitsReader* /*filter_bits_reader*/,
     Statistics* stats, const InternalKeyComparator comparator,
-    const BlockBasedTable* table, const bool index_key_includes_seq)
+    const BlockBasedTable* table, const bool index_key_includes_seq,
+    const bool index_value_is_full)
```

Comment:
I saw you renamed it to "index_value_is_delta_encoded" in many places. Why not keep it consistent?

Reply:
It was the case before and had caused programming bugs. The new name is consistent with index_key_includes_seq, in the way that the default of both of them is true.

Reply:
I had a hard time understanding what does "index value full" means. You meant it was not deeply encoded, right. I didn't think that way the first time I saw it.

Reply:
Any suggestion for rename is welcomed (without changing the default).